sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Main.scala

```diff
@@ -242,7 +262,7 @@ object Main extends IOApp {
 
     eventStream
       .take(config.payloadsTotal.toLong)
-      .zipWithScan1((0, 0)) { case (idx, el) => (el._2.length + idx._1, 1 + idx._2) }
+      .zipWithScan1((0, 0)) { case (idx, el) => (el._2.getOrElse(Seq()).length + idx._1, 1 + idx._2) }
```

Comment:
`.map(_.length).getOrElse(0)` maybe? That'd avoid allocating memory for seq/empty array. Means the same though.